UnityProject/Assets/Scripts/Construction/ConveyorBelts/ConveyorBelt.cs

```diff
@@ -317,4 +317,19 @@ public void ServerPerformInteraction(HandApply interaction)
 				});
 		}
 	}
+
+	//######################################## Multitool interaction ##################################
+
+	[SerializeField]
+	private MultitoolConnectionType conType = MultitoolConnectionType.Conveyor;
+	public MultitoolConnectionType ConType  => conType;
+
+	//[SerializeField]
```

Comment:
Supposed to be commented out?

Reply:
yeah Unity was being Dumdum with Changing the values in the code but not updating on the serialised inspector